### ISSUE TITLE
Tablets/Desktop: Allow a three column layout with fixed group list

### ISSUE THREAD

**Issue by hpoul** [authpass/authpass#267]
When there is enough horizontal space we could have the group list all the way to the left.. so we would have:

Group list | Entry List | Entry Details
------------ | ------------- | --------
 *groups* | *entries* | *details*

**Comment by Satendra124** [comment]
Hey I am a bit new to this,
I am willing to contribute to this issue can you provide more information of which UI are you taking about ?
Thanks

**Comment by hpoul** [comment]
@Satendra124 sure, when you open a password file with AuthPass this is the current view you get:

<img width="912" alt="Screen Shot 2021-09-12 at 20 17 40" src="https://user-images.githubusercontent.com/313066/132998406-c45882e1-fa55-448e-9ff3-09347542391d.png">

which as two columns. 1.) The list of passwords and 2.) the password entry details

When you press the hamburger menu on the top right, the list of groups is expanded:

<img width="912" alt="Screen Shot 2021-09-12 at 20 17 46" src="https://user-images.githubusercontent.com/313066/132998428-0856280e-2a76-4f84-bb33-f6f487fde1de.png">


now on large screens it might be useful to have this list of groups not overlaying the rest of the view, but stick to the left.. so you have three columns next to each other.

**Comment by Satendra124** [comment]
Thanks for the response!
So something like this should do?
![image](https://user-images.githubusercontent.com/60358771/132998710-3e561a59-7073-4a40-8578-2c499b84af79.png)
Please let me know for any further changes

**Comment by hpoul** [comment]
@Satendra124 yes, exactly.. something like this. It should probably be either an option or some "pin" toggle button or something.

**Comment by ayush-7607v** [comment]
@hpoul Hello,
 Can I work on this or is the issue resolved ?

**Comment by JspreetSingh** [comment]
I want to work on this project.

**Comment by hpoul** [comment]
@JspreetSingh feel free.. let me know if you need any assistance..
